app/Http/Controllers/WorkScheduleController.php

```diff
@@ -8,59 +8,59 @@
 use App\Repositories\WorkingScheduleRepository;
 use App\Repositories\LocationRepository;
 
-
 class WorkScheduleController extends Controller
 {
-    public function __construct(WorkingScheduleRepository $workingScheduleRepository, LocationRepository $locationRepository)
-    {
+    public function __construct(
+        WorkingScheduleRepository $workingScheduleRepository, 
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.SuperfluousWhitespace.EndLine</br>**Severity**: error</br>**File**: app/Http/Controllers/WorkScheduleController.php</br>Whitespace found at end of line